### PR TITLE
Main page

### PR DESCRIPTION
**Pull request by ShArPman13**
#### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related trello card link

_Describe the source of requirement, like related issue link._

#### 💡 Background and solution

_Describe the big picture of your changes here_

#### 📷 Screenshots

_Add a screenshot here if needed_

### CODE REVIEW

**Review by shutikate — APPROVED** [review]

File: src/pages/auth/SignUp/SignUpDoctor.tsx [hunk 1]
```diff
@@ -31,69 +35,195 @@ const schema = object({
...
+        const username = doctorData.user.getUsername();
+        mutation.mutate({
+          id: doctorData.userSub,
+          name: name,
+          surname: surname,
+          category: category,
+          email: username,
+          city: city,
+        });
+      } catch (err) {
```
Comment (Shutya) [inline src/pages/auth/SignUp/SignUpDoctor.tsx:94]: Why do u use catch here? I suppose that you can use mutation.isError and if it's true, show error [Snackbar ](https://mui.com/material-ui/react-snackbar/#simple-snackbars)or Alert

File: src/pages/auth/SignUp/SignUpDoctor.tsx [hunk 2]
```diff
@@ -31,69 +35,195 @@ const schema = object({
...
-      <input type="text" placeholder="city" {...register("city")} />
-      <p>{errors.category?.message}</p>
-      <input type="email" placeholder="email" {...register("email")} />
-      <p>{errors.email?.message}</p>
-      <input type="password" placeholder="password" {...register("password")} />
-      <p>{errors.password?.message}</p>
-      <button type="submit">Submit</button>
-    </form>
+    <>
+      <Box component="form" onSubmit={onSubmit} className={style.form}>
```
Comment (Shutya) [inline src/pages/auth/SignUp/SignUpDoctor.tsx:102]: Could you please show loading during sending requests?

File: src/services/AuthService.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Auth } from 'aws-amplify';
+import { Auth } from "aws-amplify";
```
Comment (Shutya) [inline src/services/AuthService.ts:1]: I suppose that you need to add to add [double quotes](https://eslint.org/docs/latest/rules/quotes) rule to eslint config

File: src/pages/auth/SignUpConfirmation/SignUpConfirmation.tsx [hunk 4]
```diff
@@ -15,32 +15,39 @@ const schema = object({
...
+      onSuccess: () => {
+        navigate("/");
+      },
+    }
+  );
 
   const onSubmit = handleSubmit(async ({ code }) => {
     try {
-      mutation.mutate(code)
+      mutation.mutate(code);
```
Comment (Shutya) [inline src/pages/auth/SignUpConfirmation/SignUpConfirmation.tsx:50]: Can you please add [semicolon rule](https://eslint.org/docs/latest/rules/semi) to eslint config?

File: src/pages/AskDoctor/AskDoctor.tsx [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
+import style from "./AskDoctor.module.scss";
 
-const About = () => {
+const AskDoctor = () => {
   return (
     <div>
-      <h1 className={style.text}>About Us</h1>
-      {/* <p className={style.text}>
+      <h1 className={style.text}>Ask the Doctor</h1>
+      <p className={style.text}>
```
Comment (Shutya) [inline src/pages/AskDoctor/AskDoctor.tsx:7]: Why not Typography for h1 and p?

File: src/App.tsx [hunk 6]
```diff
@@ -18,20 +18,26 @@ function App() {
...
           <Route path="auth">
-            <Route path="sign-up-doctor" element={
-              <QueryClientProvider client={queryClient}>
-                <SignUpDoctor />
-              </QueryClientProvider>
-            }></Route>
+            <Route
+              path="sign-up-doctor"
+              element={
+                <QueryClientProvider client={queryClient}>
```
Comment (Shutya) [inline src/App.tsx]: It's strange to wrap every route by QueryClientProvider, have already communicate with Kate that there are some issues with wrapping all App by QueryClientProvider, will try to fix them.

Comment (Shutya) [reply src/App.tsx]: Just tried on your setup, everything is working, could u please wrap the whole app with QueryClientProvider?
<QueryClientProvider client={queryClient}>
  <Routes>
    ...
  </Routes>
</QueryClientProvider>
And get rid wrapping of every route by provider.

Comment (ShArPman13) [reply src/App.tsx]: done

**Review by gutsstas — APPROVED** [review]

**Pull request merged**